uaclient/util.py

```diff
@@ -90,15 +93,20 @@ def disable_log_to_console():
     (Note that the @contextmanager decorator also allows this function to be
     used as a decorator.)
     """
+    # import pdb
+    #
+    # pdb.set_trace()
+    upro_logger = logging.getLogger("ubuntu_pro")
+    log_handlers = upro_logger.handlers
+    if not log_handlers and upro_logger.parent:
+        log_handlers = upro_logger.parent.handlers
```

Comment:
In what scenario does this happen?
After this PR, we should always configure the ubuntu_pro logger and never configure the root logger - right?

Reply:
No longer needed.